docs/software/webserver_stack.md

```diff
@@ -1,25 +1,138 @@
 - [LLMP Webserver - Lighttpd / MariaDB / PHP](https://dietpi.com/phpbb/viewtopic.php?p=1335#p1335)  
 - [LLSP Webserver - Lighttpd / SQLite / PHP](https://dietpi.com/phpbb/viewtopic.php?p=1335#p1335)  
-- [phpMyAdmin - Web interface SQL admin tool](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p54)  
-- [Certbot - Free, automated SSL cert creation and setup, allowing https.](https://dietpi.com/phpbb/viewtopic.php?p=1061#p1062)  
-- [Tomcat8 - Apache Tomcat server](https://dietpi.com/phpbb/viewtopic.php?p=4316#p4316)  
 
-### Webserver
+### Let's encrypt - Enable HTTPS / SSL
+- [Certbot - Free, automated SSL cert creation and setup, allowing https](https://dietpi.com/phpbb/viewtopic.php?p=1061#p1062)  
+
+### Webservers
 
 - [Apache2 - Feature-rich webserver](https://dietpi.com/phpbb/viewtopic.php?p=1549#p1549)  
 - [Nginx - Lightweight webserver](https://dietpi.com/phpbb/viewtopic.php?p=1549#p1549)  
 - [Lighttpd - Extremely lightweight webserver](https://dietpi.com/phpbb/viewtopic.php?p=1549#p1549)
+- [Tomcat8 - Apache Tomcat server](https://dietpi.com/phpbb/viewtopic.php?p=4316#p4316)  
+
+### Databases & Data stores
+
+- MariaDB - Open Source Relational Database
+- [phpMyAdmin - SQL Administration Tool for MariaDB](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p54)  
+- SQLite - Small, Fast and High reliable SQL database engine
+- Redis - Open Source In-memory key–value Data Store
+- InfluxDB - Open Source Time Series Database
+
+### Programming & Web development Frameworks
+
+- Python 3 & Flask - Micro web framework powered by Python
+- PHP - scripting language especially suited to web development
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
 
-!!! warning "Documentation in progress..."
+## Overview
+
+If you are you looking for an inexpensive way to operate your web servers or a dynamic websites, you could choose one of the web development stacks. They come with a complete package. Not only will you have the DietPi OS, web server, but you'll have the database software, and also the scripting language. 
+
+All stacks come with PHP caches (APCU and OPcache). The sizes of which are optimized based on a share of total memory.
+
+## LAMP Webserver - Apache2 / MariaDB / PHP
+
+LAMP stack is a popular open source web platform commonly used to run dynamic web sites and servers. It is considered by many, as the platform of choice for development and deployment of high performance web applications which require a solid and reliable foundation.
+
+![dietpi-webstack-lamp](../assets/images/dietpi-software-webstack-lamp.jpg)
+
+=== "Quick start"
+
+    **Access website:**
+
+    - url = http://192.168.0.100 (`http://<your.local.ip>` or `http://<your.host.name>`)
+    - local directory = /var/www
+
+    **Access phpinfo:**
+
+    - url = http://192.168.0.100/phpinfo.php
+
+    **Access memcache info:**
+
+    - APC/u = http://192.168.0.100/apc.php
```

Comment:
No slash required here and we have it in the wordlist: https://www.php.net/manual/en/book.apcu.php

Reply:
Changed to APCu